js/reservations.js

```diff
@@ -227,17 +231,19 @@ var Reservations = function() {
             // ADD EVENTS
             selectable: true,
```

Comment:
I think we can also disable `selectable` if the user can't add items.

```suggestion
            selectable: my.can_reserve,
```